docs/FAQ.md

```diff
@@ -31,3 +31,10 @@ Bitbucket. We believe that in time there will be plugins that will provide funct
 as well. Hopefully contributed by the community.
 
 And note that implementations of Backstage can be hosted wherever you feel suits your needs best.
+
+## Is Backstage only suitable for developer portals?
+
+No, not really. The core frontend framework could be used for building any large-scale web application where multiple teams are building separate parts of the app, but you want the overall experience to be consistent.
```

Comment:
I would invert the question, so that we can start with "yes!". First time I read it I didn't see the "only" in the question

Reply:
Good point! Fixed.